### PR TITLE
Add license information

### PR DESCRIPTION
**Pull request by joshuagl**
#### Summary

root-signing does not include any licensing information. The rest of sigstore is Apache-2.0 licensed. Assuming the same is true here, this PR adds a LICENSE file and adds SPDX license headers to all code files.
The copyright year for the license headers has been set to the year in which the file was first created.

#### Release Note

NONE

### CODE REVIEW

**Review by asraa — COMMENTED** [review]
Thank you so much!

While you're at it, could you add a COPYRIGHT.txt as well? I guess that's missing too
https://github.com/sigstore/sigstore/blob/main/COPYRIGHT.txt

**Comment by joshuagl** [comment]
> While you're at it, could you add a COPYRIGHT.txt as well? I guess that's missing too https://github.com/sigstore/sigstore/blob/main/COPYRIGHT.txt

I've pushed an additional commit with the COPYRIGHT.txt from sigstore/sigstore

**Review by asraa — APPROVED** [review]

File: COPYRIGHT.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+Copyright 2021 The Sigstore Authors.
```
Comment (Hayden-IO) [inline COPYRIGHT.txt:2]: 2022

File: cmd/tuf/app/add-delegation.go [hunk 2]
```diff
@@ -1,3 +1,18 @@
+//
```
Comment (Hayden-IO) [inline cmd/tuf/app/add-delegation.go:1]: 2022 (and all other files)

File: COPYRIGHT.txt (hunk 1 above)
Comment (asraa) [reply COPYRIGHT.txt:2]: > The copyright year for the license headers has been set to the year in which the file was first created.

I assumed

Comment (asraa) [reply COPYRIGHT.txt:2]: Ah, I guess this one can be 2022? I'm not sure. I assumed it was just "when was the dir first created"

Comment (Hayden-IO) [reply COPYRIGHT.txt:2]: I’d probably go with this year, I don’t know if you can backdate copyright.

Comment (joshuagl) [reply COPYRIGHT.txt:2]: Perhaps we should just remove the year? https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

**Pull request merged**